core/controllers/acl_decorators.py

```diff
@@ -577,6 +577,46 @@ def test_super_admin(self, **kwargs):
     return test_super_admin
 
 
+def can_delete_any_user(handler):
+    """Decorator that checks if the current user is a primary admin.
```

Comment:
Perhaps instead describe this as a decorator that verifies the current user is allowed to delete any user. The fact that only primary admins can is an implementation detail, and in fact we probably _don't_ want people using this decorator as a means to specifically check if the current user is a primary admin (they should instead use the most specific decorator needed to verify whatever permission they need to verify).

Reply:
Done.